mmv1/products/containerattached/terraform.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Cluster: !ruby/object:Overrides::Terraform::ResourceOverride
+    # Don't generate the LRO handling code because it doesn't account for regional services.
+    autogen_async: false
```

Comment:
Rather than skipping autogeneration, you should be able to override the full URL for for the Cluster resource's LRO handling in api.yaml. For example: https://github.com/GoogleCloudPlatform/magic-modules/blob/c44fabcafc20de305a583fd27a3eb94df337e1ee/mmv1/products/dialogflowcx/api.yaml#L993

Reply:
I actually did try that. The generated code is:
`url := fmt.Sprintf("https://{{location}}-gkemulticloud.googleapis.com/v1/%s", w.CommonOperationWaiter.Op.Name)`

which is obviously problematic. You'll notice also that dialogflowcx does not auto-generate operation code. I couldn't find any regionalized service that does.

Reply:
yikes; opened https://github.com/hashicorp/terraform-provider-google/issues/13337 to track.